tests/index.html

```diff
@@ -15,28 +15,29 @@
              view: document.getElementById('example')
          });
 
-        
-        
         // create an new instance of a pixi stage
         var stage = new PIXI.Container();
 
-
+        stage._inspector = {
+          type: 'stage',
+          whitelist: ['position']
+        };
```

Comment:
The test has been updated to show user code annotating an object with inspector metadata.